pages/k8s/1.19/release-notes.md

```diff
@@ -30,7 +30,56 @@ Before upgrading, please read the [upgrade notes](/kubernetes/docs/upgrade-notes
 
 ## What's new
 
+- IPv6 support
 
+This release of Charmed Kubernetes can now enable the alpha IPv6 dual-stack or
+beta IPv6-only support in Kubernetes by using IPv6 CIDRs in addition to or
+instead of IPv4 CIDRs in the Kubernetes Master charm's `service-cidr` and the
+Calico charm's `cidr` charm config.
+
+More information can be found in [Using IPv6 with Charmed Kubernetes][ipv6],
+including limitations and known issues.
+
+- CIS benchmark compliance
+
+Charmed Kubernetes is now compliant with the Center for Internet Security (CIS)
+benchmark for Kubernetes. Significant changes to the `kubernetes-master` and
+`kubernetes-worker` charms have been made to achieve this. Find more information
+about these changes, running the benchmark, and analyzing test results in the
+[CIS compliance for Charmed Kubernetes][cis-benchmark] documentation.
+
+- Authentication changes
+
+File-based authentication is not compliant with the CIS benchmark. Charmed Kubernetes
+now deploys a webhook authentication service that compares API requests to Kubernetes
+secrets. If needed, any existing entries in previous authentication files
+(`basic_auth.csv` and `known_tokens.csv`) are migrated to secrets during the
+`kubernetes-master` charm upgrade.
+
+- New Calico configuration options
+
+(description) `veth-mtu` for Calico. `ignore-loose-rpf` for Calico/Canal/Tigera Secure EE.
+
+- Ubuntu 20.04
+
+The default operating system for deployed machines is now Ubuntu 20.04 (Focal). Ubuntu 18.04 (Bionic) and 16.04 (Xenial) are still supported.
+
+- SR-IOV CNI
+
+A new SR-IOV CNI addon has been made available for Charmed Kubernetes. Using
+SR-IOV CNI, it is now possible to take network interfaces that are SR-IOV
+Virtual Functions and attach them directly to pods. For more information, see
+the new [SR-IOV CNI documentation][cni-sriov].
```

Comment:
The SR-IOV CNI documentation doesn't exist yet in this PR, but is added in a separate one: https://github.com/charmed-kubernetes/kubernetes-docs/pull/454